src/main/java/ch/njol/skript/entity/BeeData.java

```diff
@@ -0,0 +1,108 @@
+/**
+ *   This file is part of Skript.
+ *
+ *  Skript is free software: you can redistribute it and/or modify
+ *  it under the terms of the GNU General Public License as published by
+ *  the Free Software Foundation, either version 3 of the License, or
+ *  (at your option) any later version.
+ *
+ *  Skript is distributed in the hope that it will be useful,
+ *  but WITHOUT ANY WARRANTY; without even the implied warranty of
+ *  MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+ *  GNU General Public License for more details.
+ *
+ *  You should have received a copy of the GNU General Public License
+ *  along with Skript.  If not, see <http://www.gnu.org/licenses/>.
+ *
+ *
+ * Copyright 2011-2017 Peter Güttinger and contributors
+ */
+package ch.njol.skript.entity;
+
+import java.util.Random;
+
+import org.bukkit.entity.Bee;
+import org.eclipse.jdt.annotation.Nullable;
+
+import ch.njol.skript.Skript;
+import ch.njol.skript.lang.Literal;
+import ch.njol.skript.lang.SkriptParser;
+
+public class BeeData extends EntityData<Bee> {
+	
+	static {
+		if (Skript.classExists("org.bukkit.entity.Bee")) {
+			EntityData.register(BeeData.class, "bee", Bee.class, 2,
+				"no nectar bee", "happy bee", "bee", "nectar bee", "angry bee", "angry nectar bee");
+		}
+	}
+	
+	private int nectar = 0;
+	private int angry = 0;
+	
+	@Override
+	protected boolean init(Literal<?>[] exprs, int matchedPattern, SkriptParser.ParseResult parseResult) {
```

Comment:
Should be using ``ParseResult`` instead of ``SkriptParser.ParseResult``

Reply:
done, good catch!